<ui version="4.0" >
 <class>CylinderForm</class>
 <widget class="QWidget" name="CylinderForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="_fileLabel" >
          <property name="text" >
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_filename" />
        </item>
        <item>
         <widget class="QPushButton" name="browseButton" >
          <property name="text" >
           <string>Browse ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Height scale:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_heightScale" >
          <property name="decimals" >
           <number>10</number>
          </property>
          <property name="minimum" >
           <double>-99.989999999999995</double>
          </property>
          <property name="singleStep" >
           <double>0.010000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Radius Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_radiusScale" >
          <property name="decimals" >
           <number>10</number>
          </property>
          <property name="singleStep" >
           <double>0.010000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="_colorLabel" >
          <property name="text" >
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QtTools::ColorButton" name="_colorButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTools::ColorButton</class>
   <extends>QPushButton</extends>
   <header>QtTools/ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
